src/gui/editorwidgets/qgsrelationreferencewidget.h

```diff
@@ -49,8 +49,6 @@ class GUI_EXPORT QgsRelationReferenceWidget : public QWidget
     Q_PROPERTY( bool openFormButtonVisible READ openFormButtonVisible WRITE setOpenFormButtonVisible )
 
   public:
-    typedef QPair < QVariant, QgsFeatureId > ValueRelationItem;
-    typedef QVector < ValueRelationItem > ValueRelationCache;
```

Comment:
I removed these two as they were not used, is this ok for you @m-kuhn ?
there seem to be another similar definition in `QgsValueRelationFieldFormatter` which I suspect replaced this one.